pvlib/spectrum/mismatch.py

```diff
@@ -88,23 +102,26 @@ def get_am15g(wavelength=None):
     the irradiance on a flat receiver is frequently called plane-of-array (POA)
     irradiance.
 
+    .. deprecated:: 0.11
+        Use ``pvlib.spectrum.get_standard_spectrum(standard="ASTM G173-03")["global"]`` instead.
```

Comment:
The `@deprecated` decorator automatically adds a `.. deprecated:: ...` admonition to the docstring, so there is no need to do it manually as well.  See https://pvlib-python--2039.org.readthedocs.build/en/2039/reference/generated/pvlib.spectrum.get_am15g.html